### ISSUE TITLE
Last lectures

### ISSUE THREAD

**Issue by mikeizbicki** [mikeizbicki/cmc-csci181#37]
Hi everyone!

Here's are last lectures for the semester about the last part of the project: https://www.youtube.com/playlist?list=PLSNWQVdrBwobVpZBchM0DGs1U2U3UO_Yl

Part 3 of the project is the last assignment for the class, and I've tried to make it a relatively easy one so as to not give you too much work during finals time.

I've really enjoyed doing deep learning with you all, and hope to see you all around in the future :)

-- Mike

**Comment by nsamko** [comment]
Are there still office hours during finals week?

**Comment by mikeizbicki** [comment]
Yes, I'll still have office hours at the regular times.  And I'll be available at other times by appointment.
